app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,37 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            openLink(new URI("https://github.com/CIRDLES/Topsoil/wiki"));
+        } catch (URISyntaxException e) {
```

Comment:
I would prefer `ex` to `e` as I might add a Checkstyle rule for one letter variable names later on (using `i` or `j` in a loop will always be an exception).